### DISCUSSION TITLE
Slim VS Standard

### DISCUSSION THREAD

**Discussion by zkoppert** [super-linter/super-linter#4829] [General]
Found an interesting stat I wanted to share. It looks like most folks who are on the latest version prefer the slim version to the standard by about 13%. Also 6376  in 24 hours is pretty cool!!

![Screenshot 2023-11-02 at 12 29 21 PM](https://github.com/super-linter/super-linter/assets/6935431/6dbbf240-ad5b-4b34-bdb7-96eb84afdd7e)

cc/ @Hanse00 @ferrarimarco just as an FYI

**Comment by ferrarimarco** [comment]
And this was just released. I had a look at the v5.4.3 version, and the gap is even wider: 37,748 vs 19,888

**Reply by zkoppert** [reply]
Wow! I feel like it's rare to have a configuration option used that much more than the standard config. Great that people like it though!

**Reply by echoix** [reply]
Is it only builds triggered by dependabot though?

**Reply by zkoppert** [reply]
we don't know the breakdown of dependabot updates or how people are using it. I would suspect dependabot is a higher percentage of the recent release since it hasn't been up very long.
